src/main/java/nitezh/ministock/activities/widget/Bonobo_widget_data_provider.java

```diff
@@ -1,5 +1,9 @@
 package nitezh.ministock.activities.widget;
 
+import java.util.ArrayList;
+import java.util.List;
+
+import android.annotation.SuppressLint;
 import android.appwidget.AppWidgetManager;
 import android.content.Context;
 import android.content.Intent;
```

Comment:
`import android.annotation.SuppressLint; import java.util.ArrayList;
import java.util.List;`

not being used